File: baekjoon/1439_뒤집기/뒤집기.py
#실버5: https://www.acmicpc.net/problem/1439

s = input()
result_zero = 0
result_one = 0
check_zero = s.split("1")

for char in check_zero:
  if (char != ""):
    result_zero += 1
check_one = s.split("0")

for char in check_one:
  if (char != ""):
    result_one += 1
print(min(result_zero, result_one))

# for i in range(len(s)-1):
#   if s[i] != s[i+1]:
#     cnt += 1

File: baekjoon/silver/5_1439_뒤집기/뒤집기.js
//실버5: https://www.acmicpc.net/problem/1439

const s = require("fs").readFileSync("/dev/stdin").toString().trim();

let result_zero = 0;
let result_one = 0;

const check_zero = s.split("1");

check_zero.forEach((value) => {
  if (value !== "") result_zero += 1;
});
const check_one = s.split("0");
check_one.forEach((value) => {
  if (value !== "") result_one += 1;
});

if (result_one < result_zero) {
  console.log(result_one);
} else {
  console.log(result_zero);
}
